jest/setup.js

```diff
@@ -14,3 +14,24 @@ global.__reanimatedWorkletInit = jest.fn();
 
 jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper');
 jest.mock('react-native-mmkv');
+
+const mockedNavigation = jest.fn();
```

Comment:
NIT: If this is not used anywhere else use inline config. If you want to keep it consider changing the name to `navigateMock`